nitrogen/src/main/kotlin/tech/sarahgallitz/nitrogen/environment/DeviceQualifiers.kt

```diff
@@ -0,0 +1,24 @@
+package tech.sarahgallitz.nitrogen.environment
+
+import com.github.takahirom.roborazzi.RobolectricDeviceQualifiers
+
+object DeviceQualifiers {
```

Comment:
Maybe try `typealias DeviceQualifiers = RobolectricDeviceQualifiers` ? 

I think that would allow it to be exposed without the maintenance burden 🤞

Reply:
I was thinking of exposing only devices that we want to test on. Updated to `typealias`. Can be looked at in future if this control is needed.